test/compilable/betterCarray.d

```diff
@@ -15,3 +15,12 @@ int foo(int[] a, int i)
 {
     return a[i];
 }
+
+// https://issues.dlang.org/show_bug.cgi?id=17787
+version (D_betterC)
```

Comment:
Since all other predefined versions prefixed by `D_` are using the PascalCase naming convention,  perhaps we use `D_BetterC` for consistency.

Reply:
I decided that consistently calling it `betterC` was a better idea :-)

Reply:
Fair enough